includes/routes/shipping.php

```diff
@@ -150,7 +150,12 @@ function fast_shipping_calculate_packages() {
 
 	// Currently we only support 1 shipping address per package.
 	if ( count( $packages ) > 1 ) {
-		return new WP_Error( 'shipping_packages', 'Shipping package to > 1 address is not supported', array( 'status' => 400 ) );
+		// Perform address check to make sure all are the same
+		for ( $x = 1; $x < count( $packages ); $x++ ) {
+			if ( $packages[0]->destination != $packages[ $x ]->destination ) {
```

Comment:
Can you change `!=` to `!==` here? Once that's done, this will be ready to go.

Reply:
Thanks Cory fixed that also